### DISCUSSION TITLE
Why do we need a (on by default) Time Signature toobar

### DISCUSSION THREAD

**Discussion by petersampsonaudacity** [audacity/audacity#4431] [Near future]
Given that the default display mode for the **Timeline** is **Minutes and Seconds** why do we need to have an on-by-default **Time Signature** toolbar.  

For the user who just sticks with the default **Minutes and Seconds** Timeline this is really unnecessary clutter in the lower tooldock.  Especially so as it occupies the most visually prominent left end of the lower tooldock

Two possible solutions:

1) move the **Time Signature** toolbar functionality to the Timeline context menu (preferably **_only_** when **Beats and Measures** is active).

2) Auto-hide the **Time Signature** toolbar when the Timeline is **Minutes and Seconds** - and auto-unhide it when the Timeline is switched to **Beats and Measures**.

Flagging @Tantacrul @DilsonsPickles and @LWinterberg  as a design/UX issue

**Comment by Tantacrul** [comment]
There's no definitive 'good place' to put time signature / tempo stuff in Audacity 3.x. But bottom left is the most 'out of the way' place for it right now - and unlikely to impact user's existing workflows.

When we get to Audacity 4, we'll be asking users to choose the kind of layout they want upon installation. Right now, we're placing the toolbar here to see how much uptake it gets and to begin receiving user requests about it. This is a small step in the beginning of a significant widening of Audacity's purpose. We can't introduce it by making it more invisible. For now, if users don't want it, they can hide it. We need to feel our way.

I'd also add that we have significantly decluttered Audacity over the last few releases specifically so the introduction of these concepts wouldn't feel overwhelming. Moreover, we still have some significant further decluttering planned for the top bar in Audacity 3.4.

**Reply by DavidBailes** [reply]
> But bottom left is the most 'out of the way' place for it right now - and unlikely to impact user's existing workflows.

@Tantacrul . For keyboard users, bottom left is not the most 'out of the way' place - it's the initial focus when they move to the lower toolbars, and so they have to then either tab or shift+tab their way out of it to get to any of the other controls in the lower toolbar. How about bottom right?

**Comment by Tantacrul** [comment]
We will still need to make some decisions about the exact settings we have on by default. For sure, we won't have snapping enabled. We also won't have beats and measures on by default in the ruler either.

We _might_ try to prompt users to enable beats and measures in the toolbar if they enable snapping to a time signature, or something similar.

**Comment by petersampsonaudacity** [comment]
@tantacrul and @DavidBailes 

How about this layout with the more important and more frequently used Time and Selection toolbars on the left
![image](https://user-images.githubusercontent.com/50205984/227162857-d3238952-f090-4a9d-995e-ad4c48042a9c.png)
